ansible/roles/data-products-deploy/templates/common.conf.j2

```diff
@@ -140,6 +140,7 @@ cloud.container.reports="reports"
 # course metrics container in azure
 course.metrics.cassandra.sunbirdKeyspace="sunbird"
 course.metrics.cassandra.sunbirdCoursesKeyspace="sunbird_courses"
+course.metrics.cassandra.sunbirdHierarchyStore="{{ cassandra_hierarchy_store_prefix }}hierarchy_store"
```

Comment:
Take the entire variable as config instead of just part of it

Reply:
done